eth/src/main/kotlin/com/d3/eth/sidechain/util/DeployHelper.kt

```diff
@@ -29,6 +29,8 @@ import java.math.BigDecimal
 import java.math.BigInteger
 import java.util.concurrent.TimeUnit
 
+const val ENDPOINT_ETHEREUM = "eth"
```

Comment:
Why do we store endpoint data inside the `DeployHelper` file?

Reply:
Inside utility tools file, what's contradiction? Where to store it then?